plutus-script-utils/src/Plutus/Script/Utils/Scripts.hs

```diff
@@ -14,10 +30,82 @@ module Plutus.Script.Utils.Scripts
     , dataHash
     ) where
 
-import Cardano.Api qualified as Script
-import Cardano.Api.Shelley qualified as Script
+import Cardano.Api qualified as C.Api
+import Cardano.Api.Shelley qualified as C.Api
+import Cardano.Ledger.Alonzo.Language (Language (PlutusV1, PlutusV2))
+import Codec.Serialise (Serialise, serialise)
+import Control.DeepSeq (NFData)
+import Data.Aeson (FromJSON, ToJSON)
+import Data.ByteString.Lazy qualified as BSL
+import Data.ByteString.Short qualified as SBS
+import Data.OpenApi qualified as OpenApi
+import GHC.Generics (Generic)
 import Plutus.V1.Ledger.Api qualified as PV1
+import Plutus.V1.Ledger.Scripts qualified as PV1
 import PlutusTx.Builtins qualified as Builtins
+import Prettyprinter (Pretty (pretty))
+
+deriving instance Serialise Language
+deriving instance OpenApi.ToSchema Language
```

Comment:
As part of an effort to keep the dependency footprint of `plutus-script-utils` small, I think this should be moved to `plutus-ledger`.